tests/src/test/java/alluxio/shell/command/MountCommandTest.java

```diff
@@ -79,37 +80,40 @@ public void mountWithWrongArgs() throws Exception {
   public void mountWithMultipleOptions() throws Exception {
     ConfExpectingUnderFileSystemFactory factory =
         new ConfExpectingUnderFileSystemFactory("ufs", ImmutableMap.of("k1", "v1", "k2", "v2"));
-    UnderFileSystemFactoryRegistry.register(factory);
-    AlluxioURI mountPoint = new AlluxioURI("/mnt");
-    String ufsPath = "ufs://" + mFolder.getRoot().getAbsolutePath();
-    Assert.assertEquals(0, mFsShell
-        .run("mount", "--option", "k1=v1", "--option", "k2=v2", mountPoint.toString(), ufsPath));
-    FileSystemTestUtils.createByteFile(mFileSystem, "/mnt/testFile1", WriteType.CACHE_THROUGH, 20);
-    Assert.assertTrue(mFileSystem.exists(new AlluxioURI("/mnt/testFile1")));
-    URIStatus status = mFileSystem.getStatus(new AlluxioURI("/mnt/testFile1"));
-    Assert.assertTrue(status.isPersisted());
-    UnderFileSystemFactoryRegistry.unregister(factory);
+    try (Closeable mUnderFileSystemFactoryRegistryRule =
```

Comment:
`mUnderFileSystemFactoryRegistryRule` is the name for a member variable. For the usage of a local variable here, simply called it `closeable` or `c` is fine.